policy/modules/contrib/rtorrent.te

```diff
@@ -0,0 +1,95 @@
+policy_module(rtorrent, 1.0.1)
+
+########################################
+#
+# Declarations
+#
+## <desc>
+## <p>
+## Allow rtorrent to use send mails
+## </p>
+## </desc>
+gen_tunable(rtorrent_send_mails, false)
+
+## <desc>
+## <p>
+## Enable necessary permissions for rutorrent
+## </p>
+## </desc>
+gen_tunable(rtorrent_enable_rutorrent, false)
+
+attribute rtorrentdomain;
+
+attribute_role rtorrent_roles;
+roleattribute system_r rtorrent_roles;
+
+type rtorrent_t;
+type rtorrent_exec_t;
+userdom_user_application_domain(rtorrent_t, rtorrent_exec_t)
+role rtorrent_roles types rtorrent_t;
+
+########################################
+#
+# rtorrent local policy
+#
+
+type rtorrent_port_t;
+corenet_port(rtorrent_port_t)
+allow rtorrent_t rtorrent_port_t:tcp_socket name_bind;
+
+userdom_read_user_home_content_symlinks(rtorrent_t)
+
+allow rtorrent_t self:process setpgid;
+allow rtorrent_t self:netlink_audit_socket { create_socket_perms nlmsg_relay };
```

Comment:
That does not make sense. This suggests that `rtorrent` logs to audit, but its incomplete because you need access to `CAP_AUDIT_WRITE` to be able to log to audit.